### PR TITLE
[plugin.audio.radiobrowser] 1.0.0

### PR DESCRIPTION
**Pull request by segler-alex**

### CODE REVIEW

File: plugin.audio.radiobrowser/addon.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.radiobrowser" name="Radio-Browser.info" version="1.0" provider-name="segler_alex@web.de">
```
Comment (phil65) [inline plugin.audio.radiobrowser/addon.xml]: providername != email

File: plugin.audio.radiobrowser/addon.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <requires>
+    <import addon="xbmc.python" version="2.0"/>
```
Comment (phil65) [inline plugin.audio.radiobrowser/addon.xml]: must be 2.24

File: plugin.audio.radiobrowser/addon.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="main.py">
+    <provides>audio</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en">Radio-Browser</summary>
+    <summary lang="de">Radio-Browser</summary>
+    <description lang="en"><![CDATA[Directory of radio stations that uses www.radio-browser.info as source.
```
Comment (phil65) [inline plugin.audio.radiobrowser/addon.xml]: no CDATA, lease.

File: plugin.audio.radiobrowser/resources/language/English/strings.po [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: segler_alex\n"
+"Language-Team: English (http://www.transifex.com/projects/p/xbmc-addons/language/en/)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+msgctxt "#30000"
```
Comment (phil65) [inline plugin.audio.radiobrowser/resources/language/English/strings.po]: please use range 32000 - 33000

**Comment by segler-alex** [comment]
thank you for your help, please have a look at it again
greetings from vienna

File: plugin.audio.radiobrowser/addon.xml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.radiobrowser" name="Radio-Browser.info" version="1.0" provider-name="sailor">
```
Comment (phil65) [inline plugin.audio.radiobrowser/addon.xml]: please use x.y.z version scheme.

**Comment by segler-alex** [comment]
done

**Pull request merged**